queries.sql

```diff
@@ -31,4 +30,71 @@ WHERE name <> 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE (weight_kg >= '10.4') AND (weight_kg <= '17.3');
\ No newline at end of file
+WHERE (weight_kg >= '10.4') AND (weight_kg <= '17.3');
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+BEGIN;
+
+DELETE FROMv animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT DOB;
+
+UPDATE animals
+SET weight_kg = weight_kg * '-1';
+
+ROLLBACK TO SAVEPOINT DOB;
+
+COMMIT;
+
+SELECT COUNT(name)
+FROM animals;
+
+SELECT COUNT(name)
+FROM animals
+WHERE escape_attempts = '0';
+
+SELECT AVG(weight_kg)
+FROM animals;
+
+SELECT MAX(escape_attempts)
+FROM animals;
```

Comment:
- Please change this query to reflect [the given quest](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Who%20escapes%20the%20most%2C%20neutered%20or%20not%20neutered%20animals%3F). The selected data should showcase which group of animals escapes the most, neutered or not neutered animals.